client/src/components/HowYouCanHelp.module.scss

```diff
@@ -0,0 +1,3 @@
+.howYouCanHelpContainer {
```

Comment:
Interesting. Should we start slitting out stuff in `styles/globals.scss` and moving them into components?

Reply:
We chatted a bit about this in Slack, but posting here for future reference also: 
To generate these, until the gatsby-plugin-scss-typescript plugin is working again (see more [here](https://github.com/usds/justice40-tool/pull/69)) I ended up using `gatsby-plugin-scss-typescript` to generate the files and then swapping it back for `gatsby-plugin-sass`